models/kyberswap/avalanche_c/kyberswap_avalanche_c_sources.yml

```diff
@@ -1,12 +1,14 @@
 version: 2
 
 sources:
-  - name: kyber_avalanche_c
+  - name: kyber_avalanche_c_trades
```

Comment:
We need to make sure the sources name matched the name in dune V2 engine.
We can check that by running query on Dune.